test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -680,6 +684,60 @@ public void cleanupCache()
             store.cleanupCache();
     }
 
+    public void snapshot(String snapshotName)
+    {
+        snapshot(snapshotName, null, now());
+    }
+
+    public void snapshot(String snapshotName, Instant creationTime)
+    {
+        snapshot(snapshotName, null, creationTime);
+    }
+
+    public void snapshot(String snapshotName, DurationSpec.IntSecondsBound ttl)
+    {
+        snapshot(snapshotName, false, ttl, null, now());
+    }
+    
+    public void snapshot(String snapshotName, DurationSpec.IntSecondsBound ttl, Instant creationTime)
+    {
+        snapshot(snapshotName, false, ttl, null, creationTime);
+    }
+
+    public void snapshot(String snapshotName, boolean ephemeral, DurationSpec.IntSecondsBound ttl, Instant creationTime)
+    {
+        snapshot(snapshotName, ephemeral,false, ttl, null, creationTime);
+    }
+
+    public void snapshot(String snapshotName, boolean skipMemtable, DurationSpec.IntSecondsBound ttl, RateLimiter rateLimiter, Instant creationTime)
+    {
+        snapshot(snapshotName, false, skipMemtable, ttl, rateLimiter, creationTime);
+    }
+
+    public void snapshot(String snapshotName, boolean ephemeral, boolean skipMemtable, DurationSpec.IntSecondsBound ttl, RateLimiter rateLimiter, Instant creationTime)
+    {
+        ColumnFamilyStore store = getCurrentColumnFamilyStore();
+        if(store != null)
+        {
+            store.snapshot(snapshotName, null, ephemeral, skipMemtable, ttl, rateLimiter, creationTime);
+        }
+    }
+    
+    public void clearSnapshot(String keyspace)
```

Comment:
`clearSnapshotForKeyspace` is better, I would think that paramter for this method is name of the snapshot, not name of keyspace.

Reply:
This method ("clearSnapshot(String keyspace)")is not used, what I used is "void clearSnapshot(String snapshotName, String keyspace)" ,but I think some one may need it in the future so I left it here.
Besides I think we can also implement some other method like  clearSnapshot(String id) in the future too.